config/configuration.go

```diff
@@ -63,6 +63,6 @@ func LoadConfig() {
 	}
 
 	if _, err := toml.DecodeFile(configFile, &Conf); err != nil {
-		log.Fatal("Error occured while parsing %s:%s", configFile, err)
+		log.Fatalf("Error occured while parsing %s: %s", configFile, err)
```

Comment:
Just to comment that `log.Fatalf `after it prints the error, it calls `os.Exit(1)`. That is why there is no return statement